osu.Game/Screens/Select/Details/UserRatings.cs

```diff
@@ -31,15 +32,25 @@ public BeatmapMetrics Metrics
 
                 const int rating_range = 10;
 
-                var ratings = Metrics.Ratings.Skip(1).Take(rating_range); // adjust for API returning weird empty data at 0.
+                if (metrics == null)
```

Comment:
So what happens if `metrics` is non-null but there are 0 ratings? Won't the else-block fail? Shouldn't this account for this scenario?